enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1762,6 +1763,8 @@ func (ws *workspace) setup(ctx context.Context) error {
 		}
 	}
 
+	writeCommandSummary(ws.log, "Starting from a clean runner")
```

Comment:
need `else { writeCommandSummary(...) }` here I think, since the above block includes a fallback to this block if the sync fails

Reply:
Ah good catch